litho-processor/src/main/java/com/facebook/litho/specmodels/generator/BuilderGenerator.java

```diff
@@ -733,4 +768,21 @@ private static MethodSpec generateReleaseMethod(SpecModel specModel) {
         .addStatement("$L.release(this)", BUILDER_POOL_FIELD)
         .build();
   }
+
+  private static String generateInitializer(PropDefaultModel propDefaultModel,
```

Comment:
@emilsjolander Do you mean 100 chars? This line seems to be 78 chars including the leading space.